src/Toolkit/Toolkit.Maui/SymbolDisplay/SymbolDisplay.cs

```diff
@@ -137,7 +137,7 @@ private async Task UpdateSwatchAsync()
             }
         }
 
-        private double GetScaleFactor() => Window?.DisplayDensity ?? 1d;
```

Comment:
Is this right? MainDisplayInfo is the main screen. What you want is the density of the window, which could be on any display

Reply:
Yes, but in testing, Window is always null. I will update it to check Window first then fall back to main display.